common/wforce-geoip2.cc

```diff
@@ -156,7 +156,7 @@ bool WFGeoIP2DB::lookupDataValue(const ComboAddress& address, const std::vector<
     }
     for (auto& i : path) {
       if (i != nullptr)
-        delete i;
+        delete[] i;
```

Comment:
iirc delete[] is NULL safe.

Reply:
Weirdly I only added the check because I got a SEGV otherwise